src/staking/manager.ts

```diff
@@ -395,6 +447,11 @@ export class BabylonBtcStakingManager extends EventEmitter {
       unsignedUnbondingTx,
       stakingTx,
     );
+
+    this.emit(
+      StakingEventType.SIGNING,
+      SigningType.UNBONDING,
+    );
     const signedUnbondingPsbtHex = await this.btcProvider.signPsbt(
```

Comment:
Why do we do this here while we also have the `createPartialSignedBtcUnbondingTransaction` method? Maybe we could invoke it here?

Reply:
I was thinking that this method either calls `createPartialSignedBtcUnbondingTransaction` as there's overlap or just having one method.